registry/auth.go

```diff
@@ -111,6 +111,8 @@ func loginV1(authConfig *cliconfig.AuthConfig, registryEndpoint *Endpoint, facto
 		serverAddress = authConfig.ServerAddress
 	)
 
+	client.Transport = transport.NewTransport(client.Transport, DockerHeaders(nil)...)
```

Comment:
`DockerHeaders` comes from https://github.com/docker/docker/pull/13265/files#diff-2b07b8a4e93be324f246494ec743cde2L35 where a call to `HTTPRequestFactory` is made and passed down to `Login()` and then to `loginV1()` (this function)